api/v1/openapi.yaml

```diff
@@ -1978,6 +1978,12 @@ definitions:
             enum:
             - ClusterIP
             - NodePort
+          name:
+            description: Service name
+            type: string
+          namespace:
+            description: Service namespace
```

Comment:
Non-blocking: perhaps mention that these usually correspond to a Kubernetes service name / namespace in the descriptions?

Reply:
Good point!